server/mediaserver.go

```diff
@@ -865,6 +865,9 @@ func (s *LivepeerServer) HandlePush(w http.ResponseWriter, r *http.Request) {
 			return
 		}
 		params := streamParams(appData)
+		if authHeaderConfig != nil {
+			params.SegUploadTimeoutMultiplier = authHeaderConfig.TranscodeTimeoutMultiplier
```

Comment:
Any reason why this is called TranscodeTimeoutMultiplier instead of SegUploadTimeoutMultiplier? Seems like it should be the latter given that this only affects the upload timeout and not the transcode timeout.

Reply:
Good point, have updated to be consistent now